apps/test-suite/tests/Fetch.ts

```diff
@@ -159,6 +159,32 @@ export function test({ describe, expect, it, ...t }) {
       }
       expect(error).not.toBeNull();
     });
+
+    it('should abort streaming request', async () => {
+      const controller = new AbortController();
+      setTimeout(() => controller.abort(), 500);
+      let error: Error | null = null;
+      try {
+        const resp = await fetch('https://httpbin.test.k6.io/drip?numbytes=512&duration=2', {
+          signal: controller.signal,
+          headers: {
+            Accept: 'text/event-stream',
+          },
+        });
+        const reader = resp.body.getReader();
+        while (true) {
+          const { done } = await reader.read();
+          if (done) {
+            break;
+          }
+        }
+      } catch (e: unknown) {
+        if (e instanceof Error) {
+          error = e;
+        }
+      }
+      expect(error).not.toBeNull();
```

Comment:
```suggestion
      let error: unknown | null = null;
      try {
        const resp = await fetch('https://httpbin.test.k6.io/drip?numbytes=512&duration=2', {
          signal: controller.signal,
          headers: {
            Accept: 'text/event-stream',
          },
        });
        const reader = resp.body.getReader();
        while (true) {
          const { done } = await reader.read();
          if (done) {
            break;
          }
        }
      } catch (e: unknown) {
        error = e;
      }
      expect(error).not.toBeNull();
      expect(error).toBeInstanceOf(Error);
```
What do you think?

Reply:
reverted because our old jasmine does not support `toBeInstanceOf` 🙈